src/dotnet/Program.cs

```diff
@@ -42,17 +42,33 @@ public static int Main(string[] args)
         {
             DebugHelper.HandleDebugSwitch(ref args);
 
+            if (string.Equals(Environment.GetEnvironmentVariable("DOTNET_CLI_CAPTURE_TIMING"), "1"))
```

Comment:
We have an Env class that has a "get Boolean value" method. Can you use that instead so we get a consistent handling of Boolean Env vars?